### ISSUE TITLE
Add TurboSpectrum

### ISSUE THREAD

**Issue by alexji** [andycasey/smhr#243]

**Comment by andycasey** [comment]
TurboSpectrum only works (or is only meant to work) with MARCS model atmospheres.

The current pickled set of MARCS model atmospheres need updating because they don't store radiative pressure at each depth point.

I am re-pickling the MARCS models with all the necessary information for each model. I am choosing to pickle only "standard" mixtures with 1.0 Solar masses (for spherical models) and prescribed microturbulence of 1.0 km/s. 

So only photospheres matching the expression `*_m1.0_t01_st_*` will be picked.

See http://marcs.astro.uu.se/ for more details.

**Comment by alexji** [comment]
This makes sense. I think it is fine to even say we'll only use the Atlas
atmospheres with MOOG and only the MARCS models with TurboSpectrum.

**Comment by andycasey** [comment]
Agreed. We could actually just trick Turbospectrum into thinking it is dealing with MARCS atmospheres and parse properly-formatted Castelli/kurucz photospheres, but I would rather not.

**Comment by andycasey** [comment]
**Progress:**
- [x] Add support for synthesis using turbospectrum
- [x] Add support to calculate atomic abundances from equivalent widths using turbospectrum
- [x] Add support for calculating equivalent_widths from abundances using turbospectrum
- [ ] Auto-install turbospectrum with SMHr?
- [ ] Include molecules with turbospectrum
- [ ] Update line lists to include additional info required by turbospectrum
- [ ] Make isotope implementation consistent between MOOG/Turbospectrum
